backend/app/services/chatbot_service.py

```
@@ -221,16 +223,16 @@ async def update_chatbot(self, user_id: ObjectId, chatbot_id: str, chatbot_data:
                 )
                 return JSONResponse(
                     status_code=HTTPStatus.NOT_FOUND,
-                    content={"message": "Chatbot not found"},
+                    content={"message": CHATBOT_NOT_FOUND_MSG},
                 )
             
-            logger.debug(f"Applying update to chatbot")
+            logger.debug("Applying update to chatbot")
             result = await chatbot_crud.update_chatbot(chatbot_id, chatbot_data)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)